pages/views/components/esl-panel-group.njk

```diff
@@ -0,0 +1,18 @@
+---
+layout: content
+title: ESL Panel Group
+seoTitle: ESL Panel Group - custom element to group ESLPanel-s to have Tabs or Accordion behaviour
```

Comment:
"to have Tabs" is more correctly written like "and to have ESLTabs" or "and to give ESLTabs"